aladdin-container.sh

```diff
@@ -96,13 +96,13 @@ function environment_init() {
         else
             cp $HOME/.kube/config $HOME/.kube_local/$CLUSTER_NAME.config
             kubectl config set-context "$NAMESPACE.$CLUSTER_NAME" --cluster "$CLUSTER_NAME" --namespace="$NAMESPACE" --user "$CLUSTER_NAME"
-            kubectl config use-context "$NAMESPACE.$CLUSTER_NAME"            
+            kubectl config use-context "$NAMESPACE.$CLUSTER_NAME"
         fi
 
         _handle_authentication_config
 
         if $INIT; then
-            $PY_MAIN create-namespace $NAMESPACE
+            $PY_MAIN create-namespace $NAMESPACE || true
```

Comment:
@abandali 
I think we missed the `|| true` in #105 - which means this will fail when the namespace already exists, before it would just ignore that case